logging/net_appender.go

```diff
@@ -35,20 +35,32 @@ type CloudConfig struct {
 // NewNetAppender creates a NetAppender to send log events to the app backend. NetAppenders ought to
 // be `Close`d prior to shutdown to flush remaining logs.
 // Pass `nil` for `conn` if you want this to create its own connection.
-func NewNetAppender(config *CloudConfig, conn rpc.ClientConn, sharedConn bool) (*NetAppender, error) {
-	return newNetAppender(config, conn, sharedConn, true)
+func NewNetAppender(
+	config *CloudConfig,
+	conn rpc.ClientConn,
+	sharedConn bool,
+	loggerWithoutNet Logger,
```

Comment:
I passed in a real `loggerWithoutNet` to net appenders, so we could have it be a real `Sublogger` of the `rdk.networking` logger.